doc/sphinx-guides/source/container/dev-usage.rst

```diff
@@ -0,0 +1,49 @@
+Development Usage
+=================
+
+Please note! This Docker setup is not for production!
+
+.. contents:: |toctitle|
+        :local:
+
+Intro
+-----
+
+Assuming you have `Docker <https://docs.docker.com/engine/install/>`_, `Docker Desktop <https://www.docker.com/products/docker-desktop/>`_,
+`Moby <https://mobyproject.org/>`_ or some remote Docker host configured, up and running from here on. Also assuming
+you have Java and Maven installed, as you are at least about to develop code changes.
+
+To test drive these local changes to the Dataverse codebase in a containerized application server (and avoid the
+setup described in :doc:`../developers/dev-environment`), you must a) build the application container and b)
+run it in addition to the necessary dependencies.
+
+Building and Running
+--------------------
+
+To build the application image, run the following command, as described in :doc:`app-image`:
+
+``mvn -Pct clean package``
+
+Now, start all the containers with a single command:
```

Comment:
Huh? Did you test this? Because this *should* start all the dependencies, too, as configured in the compose file. Let me know if this doesn't work for you.

Reply:
Yes, works on my machine! Containers go! 🤖 🤖 🤖 🤖

Reply:
it also works for me!